.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Manual Release
+# Triggers a merge from main->release, which will then trigger a release
+# from the release branch.
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+jobs:
+  merge-to-release-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+
+      - name: Merge main -> release
+        uses: devmasx/merge-branch@master
+        with:
+          type: now
+          from_branch: main
+          target_branch: release
+          github_token: ${{ secrets.MOMENTO_MACHINE_USER_GITHUB_TOKEN }}
```

Comment:
@cprice404 : we probably need this secret too and there's another `GITHUB_TOKEN` in the next file.. not sure if they serve the same purpose and i copy pasted from node-redis but lmk if i should make this the same.